src/frontend/Komponenter/Behandling/Førstegangsbehandling/OpprettFørstegangsbehandling.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+    byggTomRessurs,
+    Ressurs,
+    RessursFeilet,
+    RessursStatus,
+    RessursSuksess,
+} from '../../../App/typer/ressurs';
+import DataViewer from '../../../Felles/DataViewer/DataViewer';
+import { BodyLong, BodyShort, Button, Heading, Label } from '@navikt/ds-react';
+import { TextLabel } from '../../../Felles/Visningskomponenter/Tekster';
+import { stønadstypeTilTekst } from '../../../App/typer/behandlingstema';
+import { FamilieDatovelger, FamilieSelect } from '@navikt/familie-form-elements';
+import { erGyldigDato } from '../../../App/utils/dato';
+import AlertStripeFeilPreWrap from '../../../Felles/Visningskomponenter/AlertStripeFeilPreWrap';
+import styled from 'styled-components';
+import { Behandlingsårsak, behandlingsårsakTilTekst } from '../../../App/typer/Behandlingsårsak';
+
+type IFagsakParam = {
+    fagsakId: string;
+};
+const Container = styled.div`
+    margin: 2rem;
+    max-width: 60rem;
+`;
+
+const SkjemaContainer = styled.div`
+    margin-top: 2rem;
+    max-width: 14rem;
+`;
+
+interface Førstegangsbehandling {
+    kravMottatt: string;
+    behandlingsårsak: Behandlingsårsak;
+    barnSomSkalFødes: BarnSomSkalFødes[];
+}
+
+const inneholderBarnSomErUgyldige = (barnSomSkalFødes: BarnSomSkalFødes[]) =>
+    barnSomSkalFødes.some(
+        (barn) =>
+            !barn.fødselTerminDato ||
+            barn.fødselTerminDato.trim() === '' ||
+            !erGyldigDato(barn.fødselTerminDato)
+    );
+
+const OpprettFørstegangsbehandling = () => {
+    const { fagsakId } = useParams<IFagsakParam>();
+    const [fagsak, settFagsak] = useState<Ressurs<Fagsak>>(byggTomRessurs());
+    const [barnSomSkalFødes, settBarnSomSkalFødes] = useState<BarnSomSkalFødes[]>([]);
+    const [kravMottattDato, settKravMottattDato] = useState<string>();
+    const [årsak, settÅrsak] = useState<Behandlingsårsak>();
+    const [feilmelding, settFeilmelding] = useState<string>();
+    const [senderInn, settSenderInn] = useState<boolean>(false);
+    const { axiosRequest } = useApp();
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        axiosRequest<Fagsak, null>({
+            method: 'GET',
+            url: `/familie-ef-sak/api/fagsak/${fagsakId}`,
+        }).then(settFagsak);
+    }, [fagsakId, axiosRequest]);
+
+    const validerSkjema = (): string => {
+        if (!årsak) {
+            return 'Mangler årsak';
+        } else if (!kravMottattDato) {
+            return 'Mangler krav mottatt dato';
+        } else if (kravMottattDato && !erGyldigDato(kravMottattDato)) {
+            return 'Ugyldig dato for krav mottatt';
+        } else if (inneholderBarnSomErUgyldige(barnSomSkalFødes)) {
+            return 'Et eller flere barn mangler gyldig dato';
+        } else {
+            return '';
+        }
+    };
+
+    const opprettBehandling = (fagsakData: Fagsak) => {
+        const feilFraSkjema = validerSkjema();
+        if (!feilFraSkjema && årsak && kravMottattDato && !senderInn) {
```

Comment:
Trenger man å validere `årsak` og `kravMottattDato` her når det valideres i `validerSkjema()`? ... Ikke at det gjør så mye

Reply:
nullable-sjekk for å få lov til å oppfylle TS-kontrakten for innsending

Reply:
Det gir mening